### ISSUE TITLE
Bower info

### ISSUE THREAD

**Issue by chilipote** [AmiliaApp/backbone-bootstrap-widgets#9]
When I run

`bower info backbone-bootstrap-widgets`

I have this error 

```
/usr/local/lib/node_modules/bower/node_modules/q/q.js:128
                    throw e;
                          ^
```

Maybe, update the Q librairy version ?

![screen shot 2015-05-21 at 10 51 48](https://cloud.githubusercontent.com/assets/3026891/7751652/1a07974e-ffa9-11e4-8e23-cc40c9dc6e52.png)

**Comment by martindrapeau** [comment]
Q seems to be a bower thing. You will have to fix that.
(one of the reasons why I truly hate bower - fixing issues with it instead
of working on real things)

**Comment by chilipote** [comment]
Due `\n`, Fixed in #10
